cards/hip-injuries-femoral-head-fracture/card.md

```diff
@@ -0,0 +1,49 @@
+---
+
+title: Hip Injuries - Femoral Head Fracture
+authors:
+    - Layla Abubshait, MD
+    - Michael Gottlieb, MD, RDMS
+    - Mary Haas, MD
+created: 2016/11/14
+updates: null
+categories:
+    - Orthopedics
+
+---
+
+# Hip Injuries: Femoral Head Fracture
+
+![](image-1.png)
+
+## Xray Views
+
+- AP pelvis
+- Lateral hip
+- +/- Judet view of hip (45-degree oblique)
```

Comment:
Does marked decode html entities? If so, `+/-` should be converted to `	&plusmn;`